R/neptune_install.R

```diff
@@ -1,7 +1,7 @@
 neptune_install <-
 function(method = c("auto", "virtualenv", "conda"),
                             conda = "auto",
-                            version = "default",
+                            version = "0.15",
```

Comment:
this will work correctly with only minor specified?

Reply:
It will try to install ` pip install neptune-client=="0.15.*"` which is proper and installs latest available (without update) `0.15.x` release :wink: